packages/hash/playwright/tests/page-creation.spec.ts

```diff
@@ -10,6 +10,8 @@ const pageName = `Test page ${pageNameSuffix}`;
 const listOfPagesSelector = 'nav header:has-text("Pages")';
 const pageTitleInputSelector = '[placeholder="A title for the page"]';
 
+const modifierKey = process.platform === "darwin" ? "Meta" : "Control";
```

Comment:
**Bonus:** fix + test modifier key. We used to press `Meta+b` etc. before, which only worked on macOS. I spotted this when inspecting Playwright snapshots from CI.

### Ubuntu / Windows before

<img width="394" alt="Screenshot 2022-01-26 at 13 23 17" src="https://user-images.githubusercontent.com/608862/151171292-ca0bb375-2cdf-4c67-814f-97a77872232c.png">

### Ubuntu / Windows after

<img width="393" alt="Screenshot 2022-01-26 at 13 24 49" src="https://user-images.githubusercontent.com/608862/151171311-30c78219-8bc8-429e-ad70-b276528b38f6.png">